crates/blockifier/src/execution/contract_class.rs

```diff
@@ -432,6 +434,13 @@ fn convert_entry_points_v1(
         .collect()
 }
 
+#[cfg(feature = "parity-scale-codec")]
+fn hashmap_to_btree(
+    hashmap: HashMap<EntryPointType, Vec<EntryPoint>>,
+) -> BTreeMap<EntryPointType, BTreeSet<EntryPoint>> {
```

Comment:
I think you can keep it stored as a Vec. No need to make it a BTreeSet internaly